src/components/Datatable/BatchActions/BatchActions.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+
+import { createPaddingSpacing } from '../../../utils/helpers';
+import { FlexContainer } from '../../FlexContainer';
+import { Button } from '../../Button';
+import { ButtonVariants } from '../../Button/Button.enums';
+import {
+  AbsoluteLinkActionKind,
+  ActionPropType,
+  ActionWithSubactions,
+  RelativeLinkActionKind,
+} from '../types/Action.types';
+import { BatchActionsProps } from './BatchActions.types';
+import { Dropdown } from '../Dropdown';
+import { Icon } from '../../Icon';
+import { SSCIconNames } from '../../Icon/Icon.enums';
+import { ColorTypes } from '../../../theme/colors.enums';
+
+const BatchActionButton = styled(Button).attrs((props) => ({
+  variant: ButtonVariants.text,
+  ...props,
+}))`
+  ${createPaddingSpacing({ horizontal: 0.8 })}
```

Comment:
for my better understanding, when should I use `createPaddingSpacing` instead of defining padding directly?

Reply:
Great question but TBH I have no clear answer for that. I hope we can deal with this dilemma once we introduce spacing scales. For now I actually would go with using `create...Spacing` functions only when you want to pass `padding` and/or `margin` prop in the component. I'll remove it from here.